### ISSUE TITLE
ServiceExceptions

### ISSUE THREAD

**Issue by danielptm** [IrinaMix/FinalProjectSpring#9]
Jag har märkt att vissa service klasser är alla logiken "wrapped" i en an try/statement some kastar en ServiceException, och vissa klasser har använt det här mönstret på samma sätt. T ex. UserService or WorkItemService har inte använt det här mönstret.

Tror ni att IssueService och TeamService är bättre designad för den här anledningen? Borde vi använda det här mönstret WorkItemService och UserService också?

**Comment by imantahriri** [comment]
Jag förstod inte varför try/catch har används i de service klasserna och faktiskt tycker att det behövs inte alls för att Spring tar hand om det åt oss. men jag sa ingenting eftersom jag klagade mycket i förrgår och tyckte att ytterligare kommentarer blir kanske otrevligt!  :))

**Comment by danielptm** [comment]
Precis, efter att jag skrev det här jag märkte samma sak. Men jag intresserad av att höra vad andra tycker.

**Comment by danielptm** [comment]
Jag tror inte att det är otrevligt alls att ställa frågan, varför gör vi så här?

**Comment by imantahriri** [comment]
Kansne jag är lite för blyg! :))
